collect_app/src/androidTest/java/org/odk/collect/android/utilities/CollectServerClientTest.java

```diff
@@ -21,19 +21,19 @@
     public void setUp() throws Exception {
         // server hangs without a response queued:
         server.enqueue(new MockResponse());
-        collectServerClient = new CollectServerClient(new HttpClientConnection(), new WebCredentialsUtils());
+        collectServerClient = new CollectServerClient(new OkHttpConnection(), new WebCredentialsUtils());
     }
 
-        @Test
-        public void httpRequests_shouldHaveUseragentHeader() throws Exception {
-            // when
-            doRequest(url("/some-path"));
+    @Test
+    public void httpRequests_shouldHaveUseragentHeader() throws Exception {
```

Comment:
Should `agent` be capitalized here for more clarity? What do you think?

Reply:
👍